src/jni/types.cpp

```diff
@@ -99,6 +99,10 @@ namespace jni {
         env.check_exceptions();
     }
 
+    void JObject::call_void_method_noexcept(Env& env, MethodId method, jvalue* args) const {
```

Comment:
Nit: I'd prefer no shortform here. To me it's not obvious by name what the method does. But I'm fine either way